install_config/aggregate_logging.adoc

```diff
@@ -1873,16 +1813,67 @@ systemctl restart docker
 ----
```

Comment:
For the above `systemctl restart docker` command, won't that affect a running OpenShift environment negatively?  Won't the user want to cordon off the node first, make sure pods are drained properly to avoid service disruption?

Reply:
I can watched logs in /var/log/containers after changed driver from journald to json-file without drained node.  No sure if there are data loss.

Reply:
@anpingli what will happen in that case is that there may still be old container logs in the journald.  However, once fluentd is restarted and has detected that docker is now using json-file, fluentd will no longer perform kubernetes metadata annotation of those log records from the journal, so they won't get fields such as kubernetes.namespace_id, and will end up in the .orphaned index.